test/jvm/io/fsq/spindle/codegen/runtime/test/ScalaConstTest.scala

```diff
@@ -9,6 +9,7 @@ class ScalaConstTest {
   @Test
   def testConstantGeneration(): Unit = {
     A.assertEquals(List(1.2, 2.1, 1.1), ConstantineConstants.LISTCONST)
+    A.assertEquals(1000000000, ConstantineConstants.LONG)
```

Comment:
This number is in range of integers so it doesn't confirm that the `L` worked. I think one more zero will do it (and then add the `L` so the test compiles)